### DISCUSSION TITLE
Unpic disables Astro conversion to webp

### DISCUSSION THREAD

**Discussion by Pavel-Sushko** [ascorbic/unpic-img#641] [Q&A]
Hey everyone, I'm not sure if this is quite an issue yet, maybe I'm simply doing something wrong, but the moment I enable unpic, my images stop being converted to webp. Is there any way around this?

For context, I am hosting the images myself and importing them through an import statement. Here is my image config: 
```js
image: {
    service: imageService({
        placeholder: 'blurhash',
        layout: 'fullWidth',
    }),
},
```

**Comment by shi-rudo** [comment]
did you solve this?

I only managed to do this by using cdn type sharp. But it is actually not supported, I wonder why it does work?

```astro
import { Image } from "@unpic/astro";
<Image cdn="sharp" src={featureImage.getDirectusUrl} alt={featureImage.getAlt(lang)}/>
```
